Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml

```diff
@@ -62,7 +62,7 @@ configuration:
   name: query
   required: false
   type: 0
-  additionalinfo: Define a query to determine which offenses to fetch. E.g., "severity >= 4 AND id > 5 AND status=OPEN".
+  additionalinfo: Define a query to determine which offenses to fetch. E.g., "severity >= 4 AND id > 5 AND status=OPEN". When configuring status="OPEN" - closed mirrored XSOAR incident won't work.
```

Comment:
I think we should take `AND status=OPEN` out of the example and just generally discourage using status in the filter
Maybe "filtering by status in the query may result in unexpected behavior when changing an incident's status"

Reply:
`additionalinfo: Define a query to determine which offenses to fetch. E.g., "severity >= 4 AND id > 5". filtering by status in the query may result in unexpected behavior when changing an incident's status.
`

Like this?